lua/xcodebuild/integrations/xcode-build-server.lua

```diff
@@ -22,12 +23,46 @@ function M.is_enabled()
   return config.enabled
 end
 
+-- Toggles whether --skip-validate-bin flag is
+-- passed to xcode-build-server's config command
+-- and updates config if needed.
+function M.toggle_bin_validation()
+  if projectConfig.settings.validateBin == nil then
+    projectConfig.settings.skipValidateBin = true
+  else
+    projectConfig.settings.skipValidateBin = not projectConfig.settings.skipValidateBin
+  end
+
+  projectConfig.save_settings()
```

Comment:
Check out also `notifications.lua` and `send_project_settings`. It would be good to print this value there as well